emu/src/opcode.rs

```diff
@@ -122,6 +125,9 @@ impl Display for ArmModeOpcode {
                      BIN: |{cond:07b}|1_0_1|{l:01b}|{offset:047b}|\n"
                 )
             }
+            ArmModeInstruction::BlockDataTransfer => {
+                format!("just skip this for now")
+            }
```

Comment:
Would it be useful to implement it now?

Reply:
This part of code is disabled right now because it creates a little confusing to me, then right now is enough to keep raw data

Reply:
Got it. I didn’t notice it was disabled.